Stripe/StripeiOSTests/STPIntentWithPreferencesTest.swift

```diff
@@ -92,4 +92,39 @@ class STPIntentWithPreferencesTest: XCTestCase {
         }
         wait(for: [expectation], timeout: STPTestingNetworkRequestTimeout)
     }
+
+    func testDeferredIntentWithPreferences() {
+        let expectation = XCTestExpectation(description: "Retrieve Deferred Intent With Preferences")
+        let client = STPAPIClient(publishableKey: STPTestingDefaultPublishableKey)
+
+        client.retrieveDeferredIntentWithPreferences { result in
+            switch result {
+            case .success(let deferredIntent):
+                XCTAssertNotNil(deferredIntent)
+                XCTAssertEqual(deferredIntent.countryCode, "US")
+                XCTAssertNotNil(deferredIntent.linkSettings)
+                XCTAssertNotNil(deferredIntent.paymentMethodSpecs)
+                
+                XCTAssertEqual(
+                    deferredIntent.orderedPaymentMethodTypes,
+                    [STPPaymentMethodType.card, STPPaymentMethodType.cashApp]
+                )
+
+                XCTAssertEqual(
+                    deferredIntent.paymentMethodTypes,
+                    STPPaymentMethod.types(from: ["card", "cashapp"])
+                )
+
+                XCTAssertEqual(
+                    deferredIntent.unactivatedPaymentMethodTypes,
+                    [STPPaymentMethodType.cashApp]
```

Comment:
Someone could activate cash app on our CI and cause this to fail 🤷

Reply:
Hmmm yeah I think it's okay to not test this one specifically.